webpack.config.extensions.js

```diff
@@ -36,112 +28,50 @@ function blockScripts( type, inputDir, presetBlocks ) {
 		.map( block => path.join( inputDir, 'blocks', block, `${ type }.js` ) )
 		.filter( fs.existsSync );
 }
-/**
- * Return a webpack config object
- *
- * @return {object} webpack config
- */
-function getWebpackConfig() {
-	const workerCount = 1;
-	const cssFilename = '[name].css';
-
-	const presetPath = path.join( __dirname, 'extensions', 'setup', 'index.json' );
-	const presetIndex = require( presetPath );
-	const presetBlocks = _.get( presetIndex, [ 'production' ], [] );
-	const presetBetaBlocks = _.get( presetIndex, [ 'beta' ], [] );
-	const allPresetBlocks = [ ...presetBlocks, ...presetBetaBlocks ];
 
-	// Helps split up each block into its own folder view script
-	const viewBlocksScripts = allPresetBlocks.reduce( ( viewBlocks, block ) => {
-		const viewScriptPath = path.join( __dirname, 'extensions', 'blocks', block, 'view.js' );
-		if ( fs.existsSync( viewScriptPath ) ) {
-			viewBlocks[ block + '/view' ] = [ viewSetup, ...[ viewScriptPath ] ];
-		}
-		return viewBlocks;
-	}, {} );
+const presetPath = path.join( __dirname, 'extensions', 'setup', 'index.json' );
+const presetIndex = require( presetPath );
+const presetBlocks = _.get( presetIndex, [ 'production' ], [] );
+const presetBetaBlocks = _.get( presetIndex, [ 'beta' ], [] );
+const allPresetBlocks = [ ...presetBlocks, ...presetBetaBlocks ];
 
-	// Combines all the different blocks into one editor.js script
-	const editorScript = [
-		editorSetup,
-		...blockScripts( 'editor', path.join( __dirname, 'extensions' ), presetBlocks ),
-	];
+// Helps split up each block into its own folder view script
+const viewBlocksScripts = allPresetBlocks.reduce( ( viewBlocks, block ) => {
+	const viewScriptPath = path.join( __dirname, 'extensions', 'blocks', block, 'view.js' );
```

Comment:
Would it work to check for `view.scss` if `view.js` doesn't exist? Would that avoid creating empty `view.js` files when only CSS is needed?

Reply:
Not sure I'm following, sorry :sweat_smile: The check on the next line makes sure that we're not creating an empty `view.js` in the bundle if there is no source file with that name. Did you mean something else?

Reply:
I meant that in situation where we want to have `view.css` but don't have any JS, we currently need to create both files, where `view.js` really just is:

```js
import 'view.scss';
```

I'm suggesting that folks could not create `view.js` and we'd go ahead and still pick `view.scss` if it exists and build `view.css`.

Reply:
It does make sense! :smile: 

I agree that the dummy JS is kinda annoying, but TBH, I'd rather tackle that separately. It's arguably a somewhat different concern from what this PR does, and might require enough thought to warrant a separate PR :slightly_smiling_face: